pyfr/plugins/base.py

```diff
@@ -63,6 +64,12 @@ def __init__(self, intg, cfgsect, suffix=None):
             raise RuntimeError('System {0} not supported by plugin {1}'
                                .format(intg.system.name, self.name))
 
+        # Check that we support this particular integrator formulation
+        if not ('*' in self.formulations
```

Comment:
Overkill to support *.  Just check for intg.formulation in self.formulations and require plugins to specify both std and dual.